Cargo.toml

```diff
@@ -314,3 +314,9 @@ overflow-checks = true
 [profile.release]
 opt-level = 3
 panic = "unwind"
+
+# --- Patch to force use the correct parity-scale-codec version ---
+[patch.crates-io]
+parity-scale-codec = { git = "https://github.com/paritytech/parity-scale-codec", rev = "334b99c9cc8def7408bcd91d1ff7bfd79f7e2915", default-features = false, features = [
```

Comment:
This shouldn't be needed, what's the error without it?

Reply:
They should do a proper release today: https://github.com/paritytech/parity-scale-codec/issues/734#issuecomment-2891507859

Reply:
Ah I see, so we need an unreleased feature.